low-level/files/opt/seagate/sspl/prerequisites/sspl-prereqs.sh

```diff
@@ -0,0 +1,320 @@
+            do_cleanup=true
+            shift ;;
+        --disable-sub-mgr)
+            disable_sub_mgr=true
+            shift ;;
+        --standalone-installation)
+            install_3rd_party_packages=true
+            shift ;;
+        -V|--sspl_version)
+            [ -z "$2" ] && echo "ERROR: SSPL version(LDR_R1/LDR_R2) not provided" && exit 1;
+            SSPL_VERSION="$2"
+            shift 2 ;;
+        -E|--env)
+            [ -z "$2" ] && echo "ERROR: Environment not provided" && exit 1;
+            ENVIRONMENT="$2"
+            shift 2 ;;
+        -N|--node)
+            [ -z "$2" ] && echo "ERROR: Node name not provided" && exit 1;
+            NODE="$2"
+            shift 2 ;;
+        -R|--rpmq_pass)
+            [ -z "$2" ] && echo "ERROR: Rabbitmq password not provided" && exit 1;
+            RPMQ_PASS="$2"
+            shift 2 ;;
+        -A|--cntrlr_a)
+            [ -z "$2" ] && echo "ERROR: Controller A IP not provided" && exit 1;
+            CNTRLR_A="$2"
+            shift 2 ;;
+        --Ap|--cntrlr_a_port)
+            [ -z "$2" ] && echo "ERROR: Controller A Port not provided" && exit 1;
+            CNTRLR_A_PORT="$2"
+            shift 2 ;;
+        -B|--cntrlr_b)
+            [ -z "$2" ] && echo "ERROR: Controller B IP not provided" && exit 1;
+            CNTRLR_B="$2"
+            shift 2 ;;
+        --Bp|--cntrlr_b_port)
+            [ -z "$2" ] && echo "ERROR: Controller B Port not provided" && exit 1;
+            CNTRLR_B_PORT="$2"
+            shift 2 ;;
+        -U|--cntrlr_user)
+            [ -z "$2" ] && echo "ERROR: Controller user name not provided" && exit 1;
+            CNTRLR_USER="$2"
+            shift 2 ;;
+        -P|--cntrlr_pass)
+            [ -z "$2" ] && echo "ERROR: Controller password not provided" && exit 1;
+            CNTRLR_PASS="$2"
+            shift 2 ;;
+        -T|--taregt_build)
+            [ -z "$2" ] && echo "ERROR: Target build not provided" && exit 1;
+            TARGET_BUILD="$2"
+            shift 2 ;;
+        --i|--bmc_ip)
+            [ -z "$2" ] && echo "ERROR: BMC IP not provided" && exit 1;
+            BMC_IP="$2"
+            shift 2 ;;
+        --u|--bmc_user)
+            [ -z "$2" ] && echo "ERROR: BMC user not provided" && exit 1;
+            BMC_USER="$2"
+            shift 2 ;;
+        --p|--bmc_pass)
+            [ -z "$2" ] && echo "ERROR: BMC password not provided" && exit 1;
+            BMC_PASS="$2"
+            shift 2 ;;
+        *)
+            echo "ERROR: Unknown option provided: $1"
+            exit 1 ;;
+        esac
+    done
+}
+
+parse_args "$@"
+
+# Cleanup
+cleanup(){
+    systemctl stop rabbitmq-server
+    yum remove -y consul rabbitmq-server cortx-utils
+    yum remove -y cortx-sspl.noarch
+}
+[ "$do_cleanup" == "true" ] && cleanup
+
+# Setup common & 3rd_party repos
+echo "Setup repos" 2>&1 | tee -a ${LOG_FILE}
+CORTX_MONITOR_BASE_URL="https://raw.githubusercontent.com/mariyappanp/cortx-monitor/EOS-15396_self_prv"
```

Comment:
Can change this after 'main' merge i guess, when this setup_repos.sh is available in master. Should not tie to user or a branch

Reply:
Yes, I need to point this to 'main' branch before the final approval.

Reply:
Ok